addons/common/XEH_PREP.hpp

```diff
@@ -46,6 +46,7 @@ PREP(fixCollision);
 PREP(fixFloating);
 PREP(fixLoweredRifleAnimation);
 PREP(fixPosition);
+PREP(garrison);
```

Comment:
Unsure if these functions belong in common or not :stuck_out_tongue_closed_eyes: